src/ssl.c

```diff
@@ -18553,6 +18553,8 @@ int wolfSSL_DH_check(const WOLFSSL_DH *dh, int *codes)
     return WOLFSSL_SUCCESS;
 }
 
+#if !defined(HAVE_FIPS) || \
```

Comment:
Will this really not fit on one line w/o spaces near parens?

Reply:
Done. Thank you Todd.